lib/cc/config/validation/json.rb

```diff
@@ -19,6 +20,12 @@ def validate
         rescue ::JSON::ParserError => ex
           errors << "Unable to parse: #{ex.message}"
         end
+
+        def validate_version
+          unless data.key?("version")
+            warnings << %{missing 'version' key. Please add `"version": "2"`}
```

Comment:
Is 2 the current version? Would it make sense to make a constant out of it so this warning is correct when it's bumped, or do we not expect it to change that often?

Reply:
Version should change really rarely. I'm inclined to leave it messy since any version bump would probably radically change the organization of this parsing/validation, so leaning on DRY now would probably just make that potential future action harder.